simplecpp.cpp

```diff
@@ -2796,7 +2796,7 @@ static std::string getFileName(const std::map<std::string, simplecpp::TokenList
         return (filedata.find(header) != filedata.end()) ? simplecpp::simplifyPath(header) : "";
     }
 
-    const std::string relativeFilename = getRelativeFileName(sourcefile, header);
```

Comment:
I just wonder why you remove this const.

Reply:
That's the `performance-no-automatic-move` check. If a variable is returned and it is declared `const` it cannot be moved but is copied instead.

Reply:
so the drawback is less safety. is a const reference a better solution perhaps?

Reply:
I will drop this for now.